agent/src/main/java/com/newrelic/agent/android/NewRelic.java

```diff
@@ -472,9 +472,12 @@ public static void recordMetric(String name, String category, int count, double
         StatsEngine.notice().inc(MetricNames.SUPPORTABILITY_API
                 .replace(MetricNames.TAG_NAME, "recordMetric"));
 
-        log.debug("NewRelic.recordMetric invoked for name " + name + ", category: " + category +
-                ", count: " + count + ", totalValue " + totalValue + ", exclusiveValue: " + exclusiveValue +
-                ", countUnit: " + countUnit + ", valueUnit: " + valueUnit);
+        if(log.getLevel() >= AgentLog.AUDIT) {
```

Comment:
You replaced a `debug()` call with `AUDIT`. Is that intended?

Reply:
yes, you mentioned it here. https://github.com/newrelic/newrelic-android-agent/issues/139#issuecomment-1753256540